edi_oca/models/edi_exchange_record.py

```diff
@@ -420,8 +420,8 @@ def _search(
             count=False,
             access_rights_uid=access_rights_uid,
         )
-        if self.env.is_superuser():
-            # rules do not apply for the superuser
+        if self.env.is_system():
+            # rules do not apply to group "Settings"
```

Comment:
Well, for security purpose I prefer not to allow EDI admins at this stage. I see no use case for this, since deleting consumer records is not expected in Production. This PR is mainly to troubleshoot and remove ghost records before going live.

When consumer record is deleted, then `model` and `res_id` are still set on `edi.exchange.record`. So no easy way to show it in UI.
Since it's corner case, I don't think we need more time on it.

Reply:
Ok, but we have the relation to the original EDI record: I would flush them or set them in a way that makes it easier to recognize that they are "orphaned".
Not a blocker here, however, could you please add a TODO? Thanks!